package.json

```diff
@@ -11,12 +11,13 @@
     "lint": "eslint .",
     "semantic-release": "semantic-release",
     "build": "hedy -v",
-    "deploy": "hedy -v --deploy --test",
-    "deploy-sequences": "hedy --no-build -no-hints -l major -l minor --cleanup-patch 1w --cleanup-minor 1m --cleanup-major 1y",
-    "deploy-ci": "hedy -v --deploy --test --pkgVersion=ci$CIRCLE_BUILD_NUM -l ci --cleanup-ci 24h"
+    "deploy": "unset HLX_AWS_ROLE; hedy -v --deploy --test",
+    "deploy-sequences": "unset HLX_AWS_ROLE; hedy --no-build -no-hints -l major -l minor --cleanup-patch=1w --cleanup-minor=1w --cleanup-major=1y --cleanup-ci=24h",
+    "deploy-ci": "unset HLX_AWS_ROLE; hedy -v --deploy --test --pkgVersion=ci$CIRCLE_BUILD_NUM -l ci --cleanup-ci=24h"
   },
   "wsk": {
     "target": "aws",
+    "awsRole": "arn:aws:iam::118435662149:role/helix-service-role-s3-rw",
```

Comment:
Why are we using a different role here?

Reply:
because only the `...s3-rw` role has write permisson on the s3. the normal role we use doesn't.
actually, we only need to `---s3-ro` role. I'm gonna change this.